examples/hyper.rs

```diff
@@ -0,0 +1,89 @@
+use hyper::{
+    service::{make_service_fn, service_fn},
+    Body, Request, Response, Server,
+};
+use prometheus_client::{encoding::text::encode, metrics::counter::Counter, registry::Registry};
+use std::{
+    future::Future,
+    io,
+    net::{IpAddr, Ipv4Addr, SocketAddr},
+    pin::Pin,
+    sync::Arc,
+    time::Duration,
+};
+use tokio::signal::unix::{signal, SignalKind};
+
+lazy_static::lazy_static! {
+    pub static ref EXAMPLE_COUNTER: Counter = Counter::default();
+}
```

Comment:
I would like to encourage users to use local metrics, thus I would prefer this example to use a locally instantiated metric.

Would you mind changing this example to use a non-global metric instead?

I am fine with having an example using global metrics in this repository, though I would like that to be a dedicated example, documenting the trade offs a user buys into.

See also https://github.com/prometheus/client_rust/issues/25